cluster/gce/list-resources.sh

```diff
@@ -75,10 +75,10 @@ echo "Provider: ${KUBERNETES_PROVIDER:-}"
 # provided.
 gcloud-compute-list instance-templates --regexp="${INSTANCE_PREFIX}.*"
```

Comment:
Since you're touching this anyway, better let's get done with this once and for all :)